modules/standard/Random.chpl

```diff
@@ -504,15 +583,92 @@ module Random {
 
 
     /*
-      Models a stream of pseudorandom numbers.  See the module-level
-      notes for :mod:`PCGRandom` for details on the PRNG used.
+
+      Models a stream of pseudorandom numbers generated by the PCG random number
+      generator.  See http://www.pcg-random.org/ and the paper, "PCG: A Family
+      of Simple Fast Space-Efficient Statistically Good Algorithms for Random
+      Number Generation" by M.E. O'Neill.
+
+      This class builds upon the :record:`pcg_setseq_64_xsh_rr_32_rng` PCG RNG
+      which has 64 bits of state and 32 bits of output.
+
+      While the PCG RNG used here is believed to have good statistical
+      properties, it is not suitable for cryptographic use. Additionally,
```

Comment:
Why call out cryptography specifically here?  What is the nature of cryptography vs. other fields that might use random numbers that makes it deserving of special mention?